samples/cpp/rag/text_rerank.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/genai/rag/text_rerank_pipeline.hpp"
+
+int main(int argc, char* argv[]) try {
+    if (argc < 4) {
+        throw std::runtime_error(std::string{"Usage: "} + argv[0] +
+                                 " <MODEL_DIR> '<QUERY>' '<TEXT 1>' ['<TEXT 2>' ...]");
+    }
+
+    auto documents = std::vector<std::string>(argv + 3, argv + argc);
```

Comment:
I think it is more convenient to specify document files that is more closer to practical use case.